app/bundles/FormBundle/Entity/Field.php

```diff
@@ -899,4 +906,20 @@ public function isFileType()
     {
         return 'file' === $this->type;
     }
+
+    /**
+     * @return bool
+     */
+    public function isLeadFieldNotOverwrite()
```

Comment:
When you'll be doing changes, could you please also add param and return types to the new methods?